proposals/22-04-localize-command.md

```diff
@@ -322,10 +314,14 @@ This command will have at least alpha and GA releases. Depending on user feedbac
 
 ### Alpha
 
-This release will ignore 
+This release will not support
 
-* local kustomization files that `target` references
+* load restrictions check
```

Comment:
qq just for clarification, we will still thrown an error if there is any referenced file outside of `scope`, right?

Reply:
Yup. I can either 

* reword "load restrictions check" to "`target` kustomizations that need `kustomize build` flag `--load-restrictor LoadRestrictionsNone` to run"
* add a note at the end of the list about checking for `scope`

Would that be helpful?

Reply:
No need to update, was just double checking. It LGTM!

Reply:
I'm also confused by this one. We aren't ever going to support Kustomizations that require that flag, are we?

Reply:
I didn't plan to check that load restrictions are followed in alpha. `kustomize localize` is guaranteed to work for Kustomizations that follow load restrictions. If the Kustomization doesn't follow load restrictions, `kustomize localize` doesn't guarantee anything, including throwing an error.

Having said that, I'm also open to implement the error-throwing, or check for Kustomizations that don't follow load restrictions.

Reply:
I was hoping that we might find a way to use the file loader as part of the implementation, so that we get the cycle detection and load restrictions for free from the start (and do not have a second implementation of them to maintain). Is that possible?

Reply:
You're 100% correct. The implementation thus far does reuse loader. I was just hesitating whether to make it an official guarantee in alpha. 

Will change the proposal. Thank you!